editoast/src/views/timetable/stdcm.rs

```diff
@@ -534,36 +535,39 @@ mod tests {
 
         let physics_consist: PhysicsConsist = simulation_parameters.into();
 
-        assert_eq!(physics_consist.mass, total_mass as u64);
+        assert_eq!(physics_consist.mass, total_mass);
 
-        assert_eq!(physics_consist.inertia_coefficient, 1.075_f64);
+        assert_eq!(
+            physics_consist.inertia_coefficient,
+            meter_per_second_squared::new(1.075)
+        );
 
         assert_eq!(
             physics_consist.rolling_resistance,
             RollingResistance {
                 rolling_resistance_type: "davis".to_string(),
-                A: 100001.0,
-                B: 1000.01,
-                C: 20.0005
+                A: newton::new(100001.0),
+                B: kilogram_per_second::new(1000.01),
+                C: kilogram_per_meter::new(20.0005),
```

Comment:
Here the values are already in N, N/(m/s), and N/(m/s)² already, so no convertion needs to be done 👍